numpy/ma/core.py

```diff
@@ -4529,28 +4558,20 @@ def iscontiguous(self):
         """
         return self.flags['CONTIGUOUS']
 
-    def all(self, axis=None, out=None):
+    def all(self, axis=None, out=None, **kwargs):
         """
-        Check if all of the elements of `a` are true.
+        Returns True if all elements evaluate to True.
 
-        Performs a :func:`logical_and` over the given axis and returns the result.
-        Masked values are considered as True during computation.
-        For convenience, the output array is masked where ALL the values along the
-        current axis are masked: if the output would have been a scalar and that
-        all the values are masked, then the output is `masked`.
+        The output array is masked where ALL the values along the current axis
```

Comment:
given axis might be more clear also maybe the ALL a bit capitalized (but I know this was already there so just noting).